packages/babel-make-styles/README.md

```diff
@@ -41,6 +41,24 @@ By default plugin handles imports from `@fluentui/react-components` & `@fluentui
 
 > NOTE: "custom-package" should re-export `__styles` function from `@fluentui/react-make-styles`
 
+### Configuring Babel settings
+
+If you need to specify custom Babel configuration, you can pass them to `babelOptions`. These options will be used by the plugin when parsing and evaluating modules.
+
+```json
+{
+  "plugins": [
```

Comment:
can we follow [standard babel preset/plugin configuration API's](https://babeljs.io/docs/en/plugins#plugin-options) ? 

```json
{
  "plugins": [
     [
       "module:@fluentui/babel-make-styles",
       {
          "plugins": ["@babel/plugin-proposal-class-static-block"],
          "presets": ["@babel/preset-typescript"]
        }
    ]

 ]
}

```

Reply:
Sorry, but I don't understand what do you mean... Plugin's configuration is represented by a tuple with a plugin name and an object options:

```json
{
  "plugins": [
    [
      "transform-async-to-module-method",
      {
        "module": "bluebird",
        "method": "coroutine"
      }
    ]
  ]
}
```

It's standard and it's the same as in our case, but we accept `modules` and `babelOptions`:

```json
{
  "plugins": [
    "module:@fluentui/babel-make-styles",
    {
      "modules": [
        { "moduleSource": "custom-package", "importName": "makeStyles" }
      ],
      "babelOptions": {
        "plugins": ["@babel/plugin-proposal-class-static-block"],
        "presets": ["@babel/preset-typescript"]
      }
    }
  ]
}
```

Are you suggesting to change the shape to a below one?

```json
{
  "plugins": [
    "module:@fluentui/babel-make-styles",
    {
      "modules": [
        { "moduleSource": "custom-package", "importName": "makeStyles" }
      ],
      "plugins": ["@babel/plugin-proposal-class-static-block"],
      "presets": ["@babel/preset-typescript"]
    }
  ]
}
```

Honestly I don't think that it will anything clearer.

Reply:
what confused me was how you wrote it within the readme docs. that's not standard definition, as it needs to be wrapped within an array.

Your docs:

```
{
  "plugins": [
    "module:@fluentui/babel-make-styles",
    {
      "babelOptions": {
        "plugins": ["@babel/plugin-proposal-class-static-block"],
        "presets": ["@babel/preset-typescript"]
      }
    }
  ]
}
```

Should been like this:

```diff
{
  "plugins": [
+  [  
    "module:@fluentui/babel-make-styles",
    {
      "babelOptions": {
        "plugins": ["@babel/plugin-proposal-class-static-block"],
        "presets": ["@babel/preset-typescript"]
      }
    }
+  ]
  ]
}
```

> Are you suggesting to change the shape to a below one?

no need, see previous comment. sorry

Reply:
Good call, it's a typo 👀 Fixed in 28b6f44.

Reply:
much better thanks!